pydkron/client.py

```diff
@@ -128,7 +128,25 @@ def get_executions(self, name):
         """
         Get the job executions for a named job
         """
-        resp = self._call(_GET, "/executions/%s" % name)
+        resp = self._call(_GET, "/jobs/%s/executions/" % name)
         if resp.status_code == 404:
             raise DkronJobNotFound("Job %s was not found" % name)
         return resp.json()
+
+    def toggle(self, name):
+        '''
+        Enable/disable a job
+
+        Arguments:
+            name {string} -- the name of job
+
+        Raises:
+            DkronJobNotFound -- raise if the job not exists
+
+        Returns:
+            DkronJob -- a job of Dkron
```

Comment:
Should the return value be a dict or a `DkronJob` in your style?

Reply:
`DkronJob` is better, I'll make an issue to tackle some of the methods that return a dict and could use `DkronJob` job instead.